catalog/models.py

```diff
@@ -178,6 +178,34 @@ class Meta:
         verbose_name=_('is popular'),
     )
 
+
+class AbstractOption(Position):
+    """
+    Product variation.
+
+    For example for product "iphone X" position can be "iphone X red".
```

Comment:
```suggestion
    "iphone X red" can be an option of "iphone X"  product  
```

Reply:
i'll still leave "For example" words and apply another sentence